go.mod

```diff
@@ -19,18 +17,17 @@ require (
 	github.com/howeyc/gopass v0.0.0-20170109162249-bf9dde6d0d2c
 	github.com/lib/pq v1.2.0
 	github.com/mitchellh/go-wordwrap v1.0.0
-	github.com/mozilla-services/yaml v0.0.0-20180922153656-28ffe5d0cafb
+	github.com/mozilla-services/yaml v0.0.0-20191106225358-5c216288813c
 	github.com/pkg/errors v0.8.1
 	github.com/sirupsen/logrus v1.4.2
-	github.com/smartystreets/goconvey v0.0.0-20190710185942-9d28bd7c0945 // indirect
 	github.com/stretchr/testify v1.3.0
 	go.mozilla.org/gopgagent v0.0.0-20170926210634-4d7ea76ff71a
+	go.mozilla.org/sops v0.0.0-20190912205235-14a22d7a7060 // indirect
```

Comment:
This seems worrying? Is there any remaining references to the old path, or does it go away with `go mod tidy`?

Reply:
@carlpett This is interesting. Running `go mod tidy` does remove it, but then `go build` adds it back. I think the changes in this PR might need to be added to master and tagged before this can be properly fixed.